app/views/users/show_remove_user.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= page_title t('users.remove_user.title') %>
+<%= link_to t('common.back'), users_path, class: 'govuk-back-link' %>
+
+<h1 class="govuk-heading-l"><%= t('users.remove_user.confirmation', user: @user.full_name ) %></h1>
+
+<%= link_to t('users.remove_user.remove_button'), remove_user_path, method: :delete, class: 'govuk-button govuk-button--warning', data: { module: "govuk-button" } %>
+
```

Comment:
Shouldn't there be a cancel button here in case the user changes their mind?

Reply:
there's a back button, so that should suffice.. I'll merge it now so we have it ready for Show&Tell today...